### PR TITLE
add Vehicle.acceleration and VehicleType.acceleration @jpshh

### PR DESCRIPTION
**Pull request by orcachillin**

### CODE REVIEW

**Comment by orcachillin** [comment]
types: https://github.com/jpxs-intl/RosaServerCore/pull/2

File: devbuild.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```
Comment (checkraisefold) [inline devbuild.sh]: get it out

**Review by checkraisefold — CHANGES_REQUESTED** [review]

File: devbuild.sh (hunk 1 above)
Comment (orcachillin) [reply devbuild.sh]: fuck forgot to ignore that

**Review by checkraisefold — APPROVED** [review]
lgtm

**Pull request merged**